graviti/openapi/file.py

```diff
@@ -57,7 +57,8 @@ def _get_upload_permission(
     sheet: str,
 ) -> Dict[str, Any]:
     key = (access_key, dataset, draft_number)
-    if int(time()) >= _PERMISSIONS.get(key, _DEFAULT_PERMISSION)["expire_at"]:
+    expire_at = _PERMISSIONS.get(key, _DEFAULT_PERMISSION)["expire_at"]
+    if expire_at == 0 or datetime.now() >= datetime.fromisoformat(expire_at.rstrip("Z")):
```

Comment:
should the default `expired_at` still be `0`

Reply:
Why `rstrip("Z")`?

Reply:
Because the ISO string in response has a "Z" at the end, which is not compatible with the input format of `datetime.fromisoformat`.